activerecord/lib/active_record/fixture_set/table_row.rb

```diff
@@ -203,6 +204,21 @@ def add_join_records(association)
             @table_rows.tables[table_name].concat(joins)
           end
         end
+
+        def serialize_attributes
+          model_class.column_names.each do |column_name|
+            if coder = attribute_coder(column_name)
+              @row[column_name] = coder.dump(@row[column_name])
+            end
+          end
+        end
+
+        def attribute_coder(column_name)
+          return if @row[column_name].nil? || String === @row[column_name]
```

Comment:
👍 , but we should probably add test coverage on these two conditions

Reply:
added. please let me know if those make sense in the way i implemented them.